test/test_issue2eslint.js

```diff
@@ -238,6 +239,13 @@ describe('issues2Eslint', function() {
             }]);
         });
 
+	it('should not filter my issue if the project config is left empty', () => {
+            const issuesObject = newIssueObject();
+            const issues = [issuesObject];
+            const filteredIssues = issues.filter(issue => issues2eslint.displayCriteria(issue, {}));
+            assert.deepEqual(issues, filteredIssues);
+	});
```

Comment:
Can you fix indentation here? :smile:

Reply:
I *think* I cleared out all the tabs here. My bad, had to muck with some editor settings coming back to JS :sweat_smile: